frontend/components/Modal/_styles.scss

```diff
@@ -77,17 +81,21 @@
     background-color: $core-white;
     padding: $pad-xxlarge;
     border-radius: 8px;
-    animation: scale-up 150ms ease-out;
+    animation: scale-up 200ms ease-out;
+    width: 100%;
 
     &__medium {
-      width: 650px;
+      max-width: 650px;
```

Comment:
Preferring `max-width` over `width` because it allows the element to resize dynamically to below that size.